### DISCUSSION TITLE
Suggest your ideas!

### DISCUSSION THREAD

**Discussion by sketchmacaw** [sketchmacaw/MacawsModsIssues#197] [Ideas]
Anyting!

**Comment by MCXiaoqing** [comment]
Add Chinese festivals in Macaw's Holiday Mod, such as Spring Festival and Lantern Festival.

**Comment by pseudonimm795** [comment]
Make it so theres a way to lock curtains on a certain configuration so they dont change when right clicked!!

**Comment by sevenlwlss** [comment]
i would absolutely LOVE if you could make it so that when you use shears on the bridges it can also only leave the railing similar to how it can only leave the platform, i recently started using mods and when using the stairs from the bridge mod i quickly noticed theres no railing or fence that fits with the railing of the bridge stairs and you cannot remove the platform from the bridge to use the railing only. would really love to see this added!!
